Updated various outdated Content Mgmt sections (#1206)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-content-view.adoc

For more information, see xref:Resolving_Package_Dependencies_{context}[].

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create Content View*.
. In the *Name* field, enter a name for the view.
{Project} automatically completes the *Label* field from the name you enter.
. In the *Description* field, enter a description of the view.

> . In the *Type* field, select a *Component Content View* or a *Composite Content View*.

Author: @maximiliankolb In my Foreman instance, I don't believe it was cherry picked because this is what I get:
![Screenshot from 2022-05-24 15-55-10](https://user-images.githubusercontent.com/91690569/170121160-fb63d042-7db0-4a56-8126-34240e94afe7.png)
I went ahead and updated Component Content View to Content View anyway.

> . In the *Type* field, select a *Component Content View* or a *Composite Content View*.

Reviewer: Yes, I have created a PR for that hoping it gets merged and accepted. As I understand it right now, this would also require a new Katello 4.3.Z release.

For more information, see xref:Resolving_Package_Dependencies_{context}[].

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create Content View*.
. In the *Name* field, enter a name for the view.
{Project} automatically completes the *Label* field from the name you enter.
. In the *Description* field, enter a description of the view.

> . In the *Type* field, choose a *Component content view* or a *Composite content view*.

Reviewer 2: Same as above.

. Optional: if you want to solve dependencies automatically every time you publish this Content View, select the *Solve Dependencies* check box.
Dependency solving slows the publishing time and might ignore any Content View filters you use.
This can also cause errors when resolving dependencies for errata.
. Click *Create Content View*.

> .Component Content View Steps

Reviewer 2: This should be treated as a separate procedure? Modularity means one procedure per .adoc

Author: A new procedure was created **modules/proc_component-content-view-steps.adoc** and added to **assembly-managing-content-views.adoc** between **proc_creating-a-content-view.adoc** and **proc_viewing-module-streams.adoc**

. Click *Create Content View* to create the Content View.
. In the *Repositories* tab, select the repository from the *Type* list that you want to add to your Content View, select the checkbox next to the available repositories you want to add, then click *Add repositories*.
. Click *Publish new version* and in the *Description* field, enter information about the version to log changes.
. Optional: You can enable a promotion path by clicking *Promote* to *Select a lifecycle environment from the available promotion paths to promote new version*.
. Click *Next*.
. On the *Review* page, you can review the environments you are trying to publish.
. Click *Finish*.

> . Optional: In the *Versions* tab after publishing, you can *Promote*, *Remove*, and *Delete* versions of the Content View under the options menu.

Reviewer: Is this unresolved?

> . Optional: In the *Versions* tab after publishing, you can *Promote*, *Remove*, and *Delete* versions of the Content View under the options menu.

Author: @maximiliankolb I removed the last Optional line.

[NOTE]

> *Remove* and *Delete* are similar but the *Delete* option will delete an entire Content View and the versions associated with that lifecycle environment. The *Remove* option allows you to choose which version you want removed from the lifecycle environment.

Reviewer 2: One sentence per line rule.

Reviewer 2: We don't use future tense as per IBM style.

Reviewer 2: We don't write "allows you" - as per Anthropomorphism section in the IBM Style guide.

Reviewer 2: s/"want removed"/"want to remove"

Author: Noted and changed per request.

> *Remove* and *Delete* are similar but the *Delete* option will delete an entire Content View and the versions associated with that lifecycle environment. The *Remove* option allows you to choose which version you want removed from the lifecycle environment.

Reviewer: Please stick to once sentence per line.

You can view the Content View in the Content Views window.
To view more information about the Content View, click the Content View name.

> To register a host to your content view, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[] in the _Managing Hosts_ guide.

Reviewer 2: ```suggestion
To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[] in the _Managing Hosts_ guide.
```

Please follow project capitalization conventions.

> To register a host to your content view, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[] in the _Managing Hosts_ guide.

Reviewer: This does not look correct.

Author: Fixed.

For more information, see xref:Resolving_Package_Dependencies_{context}[].

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.
. In the *Name* field, enter a name for the view.
{Project} automatically completes the *Label* field from the name you enter.
. In the *Description* field, enter a description of the view.

> . In the *Type* field, select a Content View or a Composite Content View.

Reviewer: ```suggestion
. In the *Type* field, select *Content View* or *Composite Content View*.
```

> . Optional: if you want to solve dependencies automatically every time you publish this Content View, select the *Solve dependencies* check box.

Reviewer: ```suggestion
. Optional: If you want to solve dependencies automatically every time you publish this Content View, select the *Solve dependencies* check box.
```

Dependency solving slows the publishing time and might ignore any Content View filters you use.
This can also cause errors when resolving dependencies for errata.

> . Optional: if you want to designate this Content View for importing from an upstream server, select the *Import only* check box.

Reviewer: ```suggestion
. Optional: If you want to designate this Content View for importing from an upstream server, select the *Import only* check box.
```

Import-only content views cannot be published directly.
. Click *Create Content View*.

.Content View Steps
. Click *Create content view* to create the Content View.
. In the *Repositories* tab, select the repository from the *Type* list that you want to add to your Content View, select the checkbox next to the available repositories you want to add, then click *Add repositories*.
. Click *Publish new version* and in the *Description* field, enter information about the version to log changes.
. Optional: You can enable a promotion path by clicking *Promote* to *Select a lifecycle environment from the available promotion paths to promote new version*.
. Click *Next*.
. On the *Review* page, you can review the environments you are trying to publish.
. Click *Finish*.

> [NOTE]

Reviewer: I suspect that you either need an empty line above or a `+` in the line above. Please check the built output.

> *Remove* and *Delete* are similar but the *Delete* option will delete an entire Content View and the versions associated with that lifecycle environment.

Reviewer: ```suggestion
*Remove* and *Delete* are similar but the *Delete* option deletes an entire Content View and the versions associated with that lifecycle environment.
```

The *Remove* option allows you to choose which version you want removed from the lifecycle environment.

You can view the Content View in the Content Views window.
To view more information about the Content View, click the Content View name.

> To register a host to your Content View, see {ManagingHostsDocURL}[Registering Hosts] in the _Managing Hosts_ guide.

Reviewer: ```suggestion
To register a host to your Content View, see {ManagingHostsDocURL}[Registering Hosts] in _{ManagingHostsDocTitle}_.
```
